src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { defineMessages, useIntl, FormattedMessage } from 'react-intl';
+
+import Icon from '~core/Icon';
+import Heading from '~core/Heading';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { Colony } from '~data/index';
+import { Address } from '~types/index';
+import { getMainClasses } from '~utils/css';
+
+import useAvatarDisplayCounter from '../../../hooks/useAvatarDisplayCounter';
+
+import styles from './VoteResultsWidget.css';
+
+interface Appearance {
+  theme?: 'approve' | 'disapprove';
+}
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.ActionsPage.VoteResultsWidget.title',
+    defaultMessage: '{title}',
```

Comment:
So why we need message descriptor for it at all?

Reply:
That's true